src/services/verify/verify.test.js

```diff
@@ -66,6 +144,21 @@ const whenGetDataReturnsDocument = () => {
   });
 };
 
+const whenGetDataReturnsDocument1 = (modifiedIssuerRecord = {}) => {
```

Comment:
Not fond of `1` as a prefix, isn't there a better naming alternative ? same for whenDnsProveResolvesAddress1

Reply:
fixed.